### PR TITLE
l10n: uk: update translation.

### PR DESCRIPTION
**Pull request by arkid15r**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -6185,9 +6184,9 @@ msgid "editing blobs is not supported"
 msgstr ""
 
 #: builtin/config.c:854
-#, fuzzy, c-format
+#, c-format
 msgid "cannot create configuration file %s"
-msgstr "неможливо записати %s файл '%s'"
+msgstr "неможливо створити конфігураційний файл %s"
```
Comment (kasya) [inline po/uk.po:6189]:
```suggestion
msgstr "неможливо створити файл конфігурації %s"
```

File: po/uk.po [hunk 2]
```diff
@@ -6331,9 +6329,8 @@ msgid "debug search strategy on stderr"
 msgstr ""
 
 #: builtin/describe.c:559
-#, fuzzy
 msgid "use any ref"
-msgstr "надіслати всі посилання"
+msgstr "використовувати будь-яке посилання"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "використати будь-яке посилання"
```

🤔

**Review by kasya — COMMENTED** [review]
LGTM

File: po/uk.po (hunk 2 above)
Comment (arkid15r) [reply po/uk.po]: Not really sure

File: po/uk.po (hunk 1 above)
Comment (arkid15r) [reply po/uk.po:6189]: 💯

**Pull request merged**